session.go

```diff
@@ -0,0 +1,113 @@
+package scylla
+
+import (
+	"fmt"
+	"net"
+	"time"
+
+	"scylla-go-driver/transport"
+)
+
+// TODO: Should we support connecting to different than default port?
+// TODO: Add retry policy.
+// TODO: Add Query Paging.
+// TODO: make better randomConnection.
+
+type SessionConfig struct {
+	hosts []string
```

Comment:
All config options shall be Exported.

Reply:
Let's add Events and Keyspace here too.

Reply:
By the way, it seems oddly to me to let somebody see what's inside the driver and what is the structure of internal packages. What is the policy of visibility in such config structs then? Should user see our internal types i.e. frame.EventType or transport.something?